### PR TITLE
Add release notes for 20.10.6

### PR DESCRIPTION
**Pull request by tiborvass**
Signed-off-by: Tibor Vass <tibor@docker.com>

### CODE REVIEW

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -22,6 +22,45 @@ for Docker Engine.
...
+* Include VPNKit binary for arm64 [moby/moby#42141](https://github.com/moby/moby/pull/42141)
+
+### Rootless
+
+* Update RootlessKit to [v0.14.1](https://github.com/rootless-containers/rootlesskit/releases/tag/v0.14.1) (see also [v0.14.0](https://github.com/rootless-containers/rootlesskit/releases/tag/v0.14.0) [v0.13.2](https://github.com/rootless-containers/rootlesskit/releases/tag/v0.13.2)) [moby/moby#42186](https://github.com/moby/moby/pull/42186) [moby/moby#42232](https://github.com/moby/moby/pull/42232)
+* dockerd-rootless.sh: prohibit running as root [moby/moby#42072](https://github.com/moby/moby/pull/42072)
+* Fix "operation not permitted" when bind mounting existing mounts [moby/moby#42233](https://github.com/moby/moby/pull/42233)
+* overlay2: fix "createDirWithOverlayOpaque(...) ... input/output error" [moby/moby#42188](https://github.com/moby/moby/pull/42188)
```
Comment (tiborvass) [inline engine/release-notes/index.md:61]: oops, this is not the cherrypick

Comment (tiborvass) [reply engine/release-notes/index.md:61]: should be https://github.com/moby/moby/pull/42235

File: engine/release-notes/index.md [hunk 2]
```diff
@@ -22,6 +22,45 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.6
+2021-04-01
```
Comment (StefanScherer) [inline engine/release-notes/index.md:26]: The date should be updated.

**Review by StefanScherer — CHANGES_REQUESTED** [review]
Only a nit, but the release date should be updated.

File: engine/release-notes/index.md [hunk 3]
```diff
@@ -22,6 +22,50 @@ for Docker Engine.
...
+* Fix implicit IPv6 port-mappings not included in API response. Before docker 20.10, published ports were accessible through both IPv4 and IPv6 by default, but the API only included information about the IPv4 (0.0.0.0) mapping [moby/moby#42205](https://github.com/moby/moby/pull/42205)
+* Fix a regression in docker 20.10, causing the docker-proxy  to not be terminated in all cases [moby/moby#42205](https://github.com/moby/moby/pull/42205)
+* Fix iptables forwarding rules not being cleaned up upon container removal [moby/moby#42205](https://github.com/moby/moby/pull/42205)
+
+### Packaging
+
+* Update containerd to [v1.4.4](https://github.com/containerd/containerd/releases/tag/v1.4.4) for static binaries. The containerd.io package on apt/yum repos already had this update out of band. Includes a fix for [CVE-2021-21334](https://github.com/containerd/containerd/security/advisories/GHSA-6g2q-w5j3-fwh4). [moby/moby#42124](https://github.com/moby/moby/pull/42124)
+* Include VPNKit binary for arm64 [moby/moby#42141](https://github.com/moby/moby/pull/42141)
+
```
Comment (chris-crone) [inline engine/release-notes/index.md:52]:
```suggestion
* Add the [Docker Scan CLI](https://github.com/docker/scan-cli-plugin) plugin

```

**Review by chris-crone — COMMENTED** [review]
One addition then LGTM

File: engine/release-notes/index.md (hunk 3 above)
Comment (thaJeztah) [reply engine/release-notes/index.md:52]: We should mention it's currently x86 only

I earlier wrote;

- Provide docker scan CLI plugin through the `docker-scan-plugin` package on Linux (x86)
- Add docker scan plugin as "recommended" dependency for the `docker-ce-cli`  package on Linux (x86)

But perhaps that's too technical; wdyt, @chris-crone ?

Comment (chris-crone) [reply engine/release-notes/index.md:52]: Yes, it'd be good to say x86 only as that impacts users

Comment (tiborvass) [reply engine/release-notes/index.md:52]:
```
* Provide the [Docker Scan CLI](https://github.com/docker/scan-cli-plugin) plugin on Linux amd64 via a `docker-scan-plugin` package as a recommended dependency for the `docker-ce-cli` package [docker/docker-ce-packaging#537](https://github.com/docker/docker-ce-packaging/pull/537)
```

**Comment by tiborvass** [comment]
@chris-crone @thaJeztah updated with reference to docker scan plugin and new distros. PTAL

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by chris-crone — APPROVED** [review]
LGTM

**Pull request merged**